Makefile.in

```diff
@@ -100,7 +100,9 @@ UTEST_MODULES = \
 	types_and_specs \
 	types_and_specs2 \
 	cuter_metrics_tests \
-	cuter_config_tests
+	cuter_config_tests \
+	examples_for_type_analysis \
```

Comment:
The module names `examples_for_type_analysis*` are misleading and confusing.  They should be changed.
There is no _type analysis_ component in this PR; just a simple conversion of the specs that these modules contain.